src/components/card/_card.scss

```
@@ -122,6 +127,15 @@
   }
 }
 
+.euiCard__hidden-bottom-link,
+.euiCard__hidden-top-link {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
```

I have found that sometimes positioning with percentages doesn't work correctly. So it's best to stick full offsets.

```suggestion
  right: 0;
  bottom: 0;
```